mandatory/2-function-creation.js

```diff
@@ -16,7 +19,18 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (num > 100) {
+    return "100%"
+  }
+  if (typeof num === "number") {
+    return `${num}%`
+  }
+  if (num % 1 != 0) {
```

Comment:
I used 2 if checks here:

- Return 2 decimals if the number is less than 1
- Return 1 decimal if the number is more than 1